src/App.tsx

```diff
@@ -1,61 +1,116 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { ChangeEvent, useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+
+function getUser(userId: number): User | null {
```

Comment:
Don't forget about past checkLists
```suggestion
function getUserById(id: number): User | null {
```